### PR TITLE
adjust red colours in the footer

### PR DESCRIPTION
**Pull request by dstodolny**
changed red colours in the footer according to: https://github.com/redbadger/website-honestly/issues/431

### CODE REVIEW

**Comment by gregdorward** [comment]
As with the Website Honestly tickets, I got the new colours the wrong way round in the last comment on the original bug, so the badgerRedOnBlack value needs to be switched with badgerRed

**Comment by gregdorward** [comment]
@dstodolny, would it be possible to revert the elements within the ul class 'category-list boldSansSerif ux-amp-design' to the original Badger Red please? It's more noticeable that these are a different shade of red to the Red Badger logo than it is with the rest of the text

Also, to ensure these pass accessibility guidelines, could we increase the font size in these elements to 18px please?

Other than those two changes, the new colours are looking good in the text

**Comment by gregdorward** [comment]
@dstodolny I've just realised that this one still needs some additional work from my previous comment when you get a chance please

**Comment by chris-brookes** [comment]
@dstodolny - The font size still needs changing.

**Comment by gregdorward** [comment]
Updated request to 18.66px, as per these guidelines - http://webaim.org/resources/contrastchecker/

**Comment by gregdorward** [comment]
After chatting to Sari, we've decided to keep the font size as is for tablet and mobile, and increase to 18.66 for desktop

**Comment by gregdorward** [comment]
Also, in order to pass accessibility checks, we need to specify that ProximaNova-Bold has bold weighting, seeing as it's a custom font

**Comment by gregdorward** [comment]
Looking good - I've sent an image to Sari to get her approval.

One final change (hopefully). Could you update the 'blog-author-name' and 'activeNavLink' elements to use the #E5051A colour please?

**Comment by dstodolny** [comment]
Any details about Sari request? I've added `font-weight: bold` to the `category-list` items already.

**Comment by gregdorward** [comment]
@sarigriffiths could you help us out with how to associate bold weight to a font without applying font-weight directly please?

File: styles/_blog.less [hunk 1]
```diff
@@ -70,7 +70,7 @@ body:not(.button-style-default) .sqs-editable-button, body:not(.button-style-def
 }
 
 .blog-author-name {
-  color: @badgerRed;
+  color: @badgerRedOnWhite;
```
Comment (gregdorward) [inline styles/_blog.less]: I think this should be 'border-bottom: 2px solid badgerRedOnWhite;', which is the case in Website Honestly

**Comment by gregdorward** [comment]
The change made on the Website-honestly ticket looks good, so could we get the same (and the border-bottom fix) applied to this one please @dstodolny?

File: styles/_blog.less (hunk 1 above)
Comment (gregdorward) [inline styles/_blog.less]: I think I've made a mistake here... this comment was supposed to be made on one of the below changes, so I think this needs reverting, sorry @dstodolny!

File: styles/_header.less [hunk 2]
```diff
@@ -34,6 +34,7 @@
 .site-header__mediumScreenNav a:hover,
 .site-header__smallScreenNavContainer a,
 .activeNavLink {
+  color: @badgerRedOnWhite;
   border-bottom: 2px solid red;
```
Comment (gregdorward) [inline styles/_header.less]: This is where I meant to leave my last comment - change to border-bottom: 2px solid @badgerRedOnWhite

This really should be the last change (honestly)

**Comment by gregdorward** [comment]
I'm happy for this one to go once it's been reviewed. I've requested some help with this from the AIMIA project team

**Comment by gregdorward** [comment]
Thanks @Etiene!

**Pull request merged**